### DISCUSSION TITLE
Experimental Deflationary Mechanism for Darwinia

### DISCUSSION THREAD

**Discussion by johneggleston** [darwinia-network/darwinia#1497] [General]
GM Darwinians! 

Over the past four years, I have deeply invested my time, resources, and passion into Darwinia, a project I believe in due to its innovative 100-year tokenomics plan aimed at ensuring sustainability. As an OG Darwinia Ambassador reapplying to the new Ambassador program, I am excited to share my plan to leverage my ambassador proceeds for a groundbreaking initiative that could significantly benefit our ecosystem.

**Proposal Summary**

As an Ambassador, I will receive $300 per month. Given the lucrative returns from native staking and liquidity farming rewards, I am committed to putting this $300 to work. I aim to demonstrate how even a relatively small amount of money can have a significant positive impact on the project. My idea is to create a mechanism that rewards the community with RING without relying on token supply inflation, instead functioning as a deflationary mechanism that builds liquidity rather than draining it.

I have recently become involved with a very obscure DeFi project called ‘Pepeito,’ the most unorthodox DeFi system I have ever seen. It uses an army of MEV bots to trade tokens within the Pepeito network, burning their profits and automatically reflecting tokens to holders. This fully decentralized and sustainable model makes the token hyper-deflationary and rewarding to holders, with bots and mechanisms that can operate indefinitely as long as the blockchain exists.

### Proposal Details

**1. Allocation of Ambassador Funds:** I propose to allocate 100% of my ambassador funds ($300 per month) to the accumulation of these experimental assets for the Darwinia treasury. Each month, the funds would be used to:

    * Purchase $300 worth of Pepeito tokens.
    
    * Give away $100 worth to the Darwinia community.
    
    * Keep the remaining $200 worth in the treasury to build over time. 

**2. Deflationary Mechanism:** As the position grows into a sizeable sum, we can set aside the base capital amassed and use reflections to buy RING, burn half that RING, and distribute the other half to Darwinia community members. If this model proves successful, we can automate the process through a smart contract that uses reflections to buy RING, burn half, and distribute half to RING liquidity providers.

**3. Stable Deflation and Rewards:** This mechanism aims to create stable deflation and rewards for the Darwinia ecosystem. While the idea is experimental and its outcome is uncertain, the evidence from the Pepeito community, though small (around 10 active members), shows consistent positive results over the years, with bots burning up to 95% of the supply on some tokens.

**Promotion and Community Engagement**

Financial benefits aside, I will actively champion Darwinia in all my promotion efforts for Pepeito. Additionally, I propose hosting giveaways of Darwinia tokens to the Pepeito community as it builds up and we secure more partners.

**Additional Details About Pepeito**

* Pepeito is a unique DeFi project with no website, whitepaper, or social campaigns, relying solely on bots for sustainability over the past three years.

* The developer, known as ‘Pepeito’ on Telegram (@Pepeito1), is highly active and responsive. The project's Telegram group, 'Severance' (t.me/mevbot), has 160 members, with around 10 actively engaged.

* The developer previously worked on a similar public project called ‘Ottobought’ before rebuilding Pepeito from the ground up, enhancing it significantly.

* Pepeito tokens exist on multiple chains, including BSC, Arbitrum, Solana, and Base, with separate versions for each chain.

Here is a link to the dextools chart for the Pepeito token on BSC. Please zoom out to the daily chart and check the holdings of the burn address, with 99% of activity carried out by bots:
[Pepeito (BSC) Dextools Chart](https://www.dextools.io/app/en/bnb/pair-explorer/0x920b38aeedb886466b12b88106daad410eb9c874?t=1716050018120)

**Conclusion**

This proposal is experimental and no outcome is certain. However, the evidence from the Pepeito project suggests promising potential. By implementing this mechanism, we can create stable deflation and rewards for the Darwinia ecosystem, enhancing liquidity and community engagement. I look forward to discussing the feasibility, potential challenges, and necessary steps to establish this innovative mechanism for Darwinia's growth.

Please ask me any questions you like! I am very excited to know everyone’s thoughts! 

Cheers,
John.

**Comment by JuneCribb** [comment]
The community might be interested seeing that the funds are yours not theirs considering how uncertain and risky this looks. 

But you put out a good proposal and this is crypto where we take risks for the greater good. 

Lovely John

**Reply by johneggleston** [reply]
Thanks June. Just want to clarify that there is no risk. It’s 100% my funds, and 100% going in the Darwinia treasury. Uncertainty shouldn’t be a concern as the result will be net positive no matter what it is. 👍

**Comment by Laki389** [comment]
Hi John, I'll be honest and direct as always.

The opening post in this discussion looks like a shill.
( on a project (s..tcoin) with 3k mcap and 1$ daily trading volume, as I could see from the link provided above)

![Screenshot_1](https://github.com/darwinia-network/darwinia/assets/133054568/ffdd2170-8245-4e8e-82ab-79bd525ea249)

**Reply by johneggleston** [reply]
Hey Laki, thanks for bringing this up because I got the link wrong, I have now edited the link so that it’s for Pepeito on BSC which is here: https://www.dextools.io/app/en/bnb/pair-explorer/0x920b38aeedb886466b12b88106daad410eb9c874?t=1716050018120

$55k MC as of writing and $5 24h volume. The reason for the low volume at this point in time is because the bots are programmed to react to market heat, which for BSC was a couple months ago. Now it’s hot on Solana. 

I’m also going to be very straightforward about something myself, I firmly believe that cross-project support is vital for the success of smaller crypto projects like Darwinia. I get frustrated seeing heaps of solid projects I’m invested in miss out on so many cool opportunities to use the tech being built with other projects, to further strengthen their own project and community. The projects I talk about openly on this platform like Pepeito and Hyperbolic Protocol are not competitors of Darwinia, and I don’t care if people invest in those projects themselves or not, but there are so many potential synergies that it’s mind-bending. 

It’s because of the reservations that come with such unusual proposals that I have decided to use my own funds from the Ambassador program. I have zero expectation for the Darwinia community to buy Pepeito, that’s what the bots are for 😉.

**Reply by Laki389** [reply]
> Hey Laki, thanks for bringing this up because I got the link wrong, I have now edited the link so that it’s for Pepeito on BSC which is here: https://www.dextools.io/app/en/bnb/pair-explorer/0x920b38aeedb886466b12b88106daad410eb9c874?t=1716050018120

I checked the new link you edited,
there are 5 alerts for that project, like:

- Buy Tax: 10%
- Sell Tax: 8.1% - 9.97%
- Tax Modifiable: Yes (This means that whoever controls the contract can put a 99% sell tax, for example.)
- Whitelisted: Yes (if there is a whitelist, some addresses may not be able to trade normally)
- Other Risks: Yes

If I were you, I would not put funds in that project.
Do you have any connection with them?
In the sense, just a sympathizer/hodler or something more?

**Reply by johneggleston** [reply]
Ahhhh yes. I am aware of those alerts. First I’ll elaborate on my role in the ecosystem, then I’ll address the alerts. 

Firstly, I have no official affiliation with the project. The founder holds a firm belief that if a project is truly decentralised, then anyone can come in and take on any role they like regardless of his opinion. This was striking to me because this is an attitude that would only be held by a man who is very serious about the principle of decentralisation. 

Basically what happened was I stumbled upon this ecosystem whilst having a browse through some exotic token pairings with big meme coins on Pancakeswap, and noticed a couple of tokens that had low market cap, but lots of transactions with the big meme coin pairing, so I decided to check it out. 

I was so bewildered by what I found that I decided to personally do a lot of digging and since then have been left without a shadow of a doubt that this system is something special and that I have found a needle in a haystack. 

The founder has taken these special tokens in the ecosystem and plugged them into rugged projects in order to revive them. He has also created a token called ‘BurnLUNA’ which the bots trade and use profits to burn Classic Luna and reflect Classic Luna onto BurnLUNA holders. 

I have taken the personal initiative to try something myself - plugging strong, legitimate projects that I am involved with into this ecosystem to see just how powerful this new dimension of DeFi can be.  

And FYI, the founder is partially doxxed and is an American citizen, so it’s definitely safe. I can provide more info over a call if you like! 

Now onto your concerns about the alerts… 

**The Taxes**

The taxes may seem a little dodgy at first, but unlike most taxed tokens, you can see the tokens you pay as taxes being put to work straight away. I have some Pepeito in a couple of wallets, and when I make a purchase with one, I can immediately see some of the tax reflection in my other wallet. So the taxes are reflected onto all holders (and that includes the bots). 

Another thing the taxes do is deter other outsider bots from trading against the bots that are trading for the Pepeito ecosystem. 

Not all the tokens in the Pepeito ecosystem are taxed though, as the developer is always experimenting with new approaches and that includes working with untaxed tokens. 

Check out the another token in the Pepeito ecosystem on Solana called SolDoge here: https://www.dextools.io/app/en/solana/pair-explorer/AkMm1Rj7y3avt84Kac59gsxhGgH9jgWUNpyenctxAP46?t=1716171127869

**Contract Ownership/Whitelist**

Yes, the dev owns the contract to the Pepeito contract but not all the contracts of the tokens in the Pepeito ecosystem. The contract ownership is good in this case it’s it allows him to continue experimenting and improving upon the system. 

As for the whitelist, the role it plays is purely to make the bots more efficient/set limits for them/etc.

**Reply by Laki389** [reply]
> Ahhhh yes. I am aware of those alerts. First I’ll elaborate on my role in the ecosystem, then I’ll address the alerts.

All right, @johneggleston 
Of course, you can do whatever you want with your own funds.

**Reply by johneggleston** [reply]
Cheers, I just thought there would be a lot of questions about what I was doing so I wanted to make sure everybody would be on board with this before I kicked things off!

**Comment by SasoLithops** [comment]
Hi John!

I am not the one to tell you how to invest your money, but since you started debate on this I will pass an advice.

Since main rule in crypto is to spread your eggs in more baskets, I would propose you spread 300$ among two or three projects. You keep them for three-six months then reward community. To not burden yourself with airdrop costs.

WHY?

- Investing all in one such low cap project can prove to be very risky. I read once 299 out of 300 low cap crypto token investments end badly(rug, abandonment, no exit liquidity....)
- Not knowing the token, community might not interact with its contract

I know I sound very pesimistic but all speaking out of massive experience. Of course this token could 1000x and I would be totally wrong. Has happened too.

So at the end it is your investment and up to you alone to do what you want with it. All I can say is thank you for out of the box thinking:-)

**Reply by johneggleston** [reply]
Thank you Saso, this actually leads me to one of the best parts. Because the market caps of the tokens in this network are so low, I can actually match the liquidity provision myself without too much of a heavy investment. This means that if the founder did remove liquidity and “rugged” the project, my provided liquidity would be enough to sustain the project. 

As for the question of “but what happens when the founder stops support completely?” - That doesn’t really matter because these tokens already have everything they need to grow built in. 

As for putting all my eggs in one basket, as someone who has been in crypto for over 5 years now. I can say that the strategy of identifying and ‘Sniping’ a small number of strong low caps has been a strategy that has worked very well for me. In the last bullrun I identified a strong nano-cap and invested $10k. This $10k turned into over $2m at its peak (over 200x growth). 

It was a $500k MC project but it had a strong founder in the military who had been building blockchain products for 6 years, it had awesome DeFi features, it’s own native L1 blockchain, and most importantly, an ecosystem that encouraged project development lead by community members instead of just the founder/team. 

And the best part is, this project I’m talking about is far more impressive on all fronts. 

This is far from my first rodeo, I know what I’m doing.

**Reply by SasoLithops** [reply]
John thanks for detail explanation. It feels now that you explained it, you have done great research and are highly aware of all the risk and have experiences in low market cap coins.

All I can say is thank you for taking on that risk for community with your own funds.

Looking forward on how this plays out.
This is definitely one of the most interesting ideas I ever heard👏👏 

P.S. Huge congrats on nano cap 200x. Life changing investment😎

**Comment by hackfisher** [comment]
I like deflationary concepts but I lack understanding of Pepeito. For further discussions about deflationary mechanisms, please move to: https://github.com/orgs/ringecosystem/discussions/4.
